plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/plugin.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+
    <extension point="org.eclipse.jdt.ui.classpathContainerPage">
      <classpathContainerPage
        id="com.google.cloud.tools.eclipse.appengine.libraries.ui"
        name="App Engine Libraries"
        class="com.google.cloud.tools.eclipse.appengine.libraries.ui.AppEngineLibrariesPage">
     </classpathContainerPage>
+    <classpathContainerPage
+       id="com.google.cloud.tools.eclipse.appengine.libraries.ui"
```

Comment:
Ah ha, according to the doc, it looks like the ID should probably be `com.google.cloud.tools.eclipse.appengine.libraries`. It is [the ID used by our `classpathContainerInitializer`](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml#L104).

Indeed, using it clears the warning:

![selection_041](https://cloud.githubusercontent.com/assets/10523105/23874576/931aa5be-080c-11e7-98e1-1bef14760f8c.png)

![selection_042](https://cloud.githubusercontent.com/assets/10523105/23874607/b03de728-080c-11e7-9d18-860980ec761f.png)